src/Info.php

```diff
@@ -27,4 +27,11 @@ class Info
         CONSOLE;
 
     public const TRAP_ROOT = __DIR__ . '/..';
+    public const JOKES = [
+        'Why do programmers always mix up Halloween and Christmas? Because Oct 31 == Dec 25.',
+        'Two hard things in computer science: cache invalidation, naming things and stack overflow.',
+        'Depressive programming style through dump and die.',
+        'PHP was dead 84 years ago right?',
+        'Submit a pull request to help us improve the Buggregator Trap codebase',
+    ];
```

Comment:
Please move it into `resources/registry/jokes.txt` file.
Format: base64 encoded joke text per line.

Why encoded?
- not to spoiler all the messages
- support multiline texts

It's OK not to make a service, and just implement a reader inside the command.

Reply:
done